requirements.txt

```diff
@@ -6,4 +6,5 @@ Flask-SQLAlchemy
 redis
 APScheduler
 python-dotenv
-eventlet
\ No newline at end of file
+eventlet
+flask_wtf
```

Comment:
write the library name as according to the docs: `Flask-WTF`
https://flask-wtf.readthedocs.io/en/0.15.x/install/

not sure if `flask_wtf` refers to the same thing